packages/sfui/frameworks/vue/components/SfGroup/SfGroup.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+
+defineProps({
+  invalid: {
+    type: Boolean,
+    default: false,
+  },
+  legend: {
+    type: String,
+    default: 'Label',
+  },
+  hintText: {
+    type: String,
+    default: 'Hint',
+  },
+  errorText: {
+    type: String,
+    default: 'Error message',
+  },
+});
+
+</script>
+
+<template>
+<fieldset>
+  <legend>{{ legend }}</legend>
```

Comment:
You'd like to eventually style the legend I guess, so we'd have to pass classes to it.

Reply:
sure, this was ony to show the main idea without stying and adding classes but you right thers should be this possibility